src/components/interfaces/HMI_API.xml

```diff
@@ -2267,6 +2270,25 @@
       </param>
   </struct>
 
+  <struct name="SpatialStruct">
```

Comment:
`x`, `y`, `width`, and `height` parameters are moved to a separate struct `Rectangle`. Descriptions have been changed for these parameters.